migrations.json

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
do we need to commit this? not sure

Reply:
It looks like a lock file for nx migration.
In Brikl they commit it so I followed them.

Reply:
i thought nx migration is just changing the source code in the git already, so i wonder why we're committing this too. Do we have to run the migration again in the future?

Reply:
🤔🤔🤔 Good question, I don't know 😂. Let's wait for some nx expert to answer this.

Reply:
No Nx expert in Thinc.. Can you ask Brikl members?